data/formats-data.js

```diff
@@ -7155,7 +7155,7 @@ let BattleFormatsData = {
 		doublesTier: "DUU",
 	},
 	buzzwole: {
-		randomBattleMoves: ["superpower", "leechlife", "stoneedge", "poisonjab", "earthquake"],
+		randomBattleMoves: ["superpower", "drainpunch", "leechlife", "stoneedge", "poisonjab", "earthquake", "bulkup", "roost"],
```

Comment:
I don't like this change at all because Scarf/Band is far more optimal. You're making what is imo one of the top random mons into something average.